Exiled.API/Features/Player.cs

```diff
@@ -925,28 +937,24 @@ public static Player Get(string args)
             }
         }
 
-        /// <summary>
```

Comment:
This should be added back with a deprecation notice. Any plugins using it will break.